css/properties.json

```diff
@@ -4733,6 +4749,54 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/field-sizing"
   },
+  "fill": {
+    "syntax": "none | <color> | <url> [none | <color>]? | context-fill | context-stroke",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "byComputedValueType",
```

Comment:
Should we be listing:

>the fill painting properties—[fill-color](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-color), [fill-image](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-image), [fill-origin](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-origin), [fill-position](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-position), [fill-size](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-size), and [fill-repeat](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-repeat)

Similar to https://github.com/mdn/data/pull/657/files?

Reply:
no. 
`<paint> is defined as color and fallback, not background image.
[https://svgwg.org/svg2-draft/painting.html#FillProperties is ](https://svgwg.org/svg2-draft/painting.html#SpecifyingPaint)

origin, position, size, and repeat are not part of paint in any implementation